libs/etterlatte-ktor/src/main/kotlin/HealthRoute.kt

```diff
@@ -0,0 +1,20 @@
+package no.nav.etterlatte.libs.ktor
+
+import io.ktor.http.HttpStatusCode
+import io.ktor.server.application.call
+import io.ktor.server.response.respond
+import io.ktor.server.routing.Route
+import io.ktor.server.routing.get
+import io.ktor.server.routing.route
+
+fun Route.healthApi() {
+    route("health") {
```

Comment:
Har vi blitt enige om `health`? Hele søknadsappen bruker `internal`, og jeg ser deler av sak også gjør det. Tror det er en fordel om vi bestemmer oss for én av de.

Reply:
Trenger vi noen av de to prefiksene? Ser flere uten dem og.

Reply:
Ser ut til å være litt av hvert her ja,
```
/internal
/health
/
```
Har dere noen preferanser? Ser at R&R serverer dem på `/`. Usikker på hva som egentlig skjer hvis vi server de samme endepunktene som R&R?

Reply:
Går for health inntil videre